server/komodo-rest/src/main/java/org/komodo/rest/service/KomodoTeiidService.java

```diff
@@ -2142,4 +2370,38 @@ public Response ping(final @Context HttpHeaders headers,
             return createErrorResponseWithForbidden(mediaTypes, e, RelationalMessages.Error.TEIID_SERVICE_QUERY_ERROR);
         }
     }
-}
+    
+    /*
+     * Loads default translator mappings from resource file
+     */
+    private void loadDefaultTranslatorMap() {
+        InputStream inputStream = getClass().getClassLoader().getResourceAsStream( DEFAULT_TRANSLATOR_MAPPING_FILE );
+        
```

Comment:
Do you want to rely on the ArgCheck in FileUtils.streamToString() for testing if inputStream is null? Its very unlikely to happen so I will not hold up the merge with it but possibly add in a null check and error message?